ArmVirtPkg/ArmVirtQemu.dsc

```diff
@@ -502,16 +502,18 @@
   #
 !include NetworkPkg/NetworkComponents.dsc.inc
 
+!if $(NETWORK_ENABLE) == TRUE
   NetworkPkg/UefiPxeBcDxe/UefiPxeBcDxe.inf {
     <LibraryClasses>
       NULL|OvmfPkg/Library/PxeBcPcdProducerLib/PxeBcPcdProducerLib.inf
   }
 
-!if $(NETWORK_TLS_ENABLE) == TRUE
-  NetworkPkg/TlsAuthConfigDxe/TlsAuthConfigDxe.inf {
-    <LibraryClasses>
-      NULL|OvmfPkg/Library/TlsAuthConfigLib/TlsAuthConfigLib.inf
-  }
+  !if $(NETWORK_TLS_ENABLE) == TRUE
+    NetworkPkg/TlsAuthConfigDxe/TlsAuthConfigDxe.inf {
+      <LibraryClasses>
+        NULL|OvmfPkg/Library/TlsAuthConfigLib/TlsAuthConfigLib.inf
+    }
+  !endif

```

Comment:
Please be consistent with the indentation: below, there is a nested NETWORK_ENABLE/NETWORK_IP4_ENABLE conditional where the indentation is untouched, whereas here, you indent the inner conditional along with the lines it guards.

I prefer the NETWORK_ENABLE/NETWORK_IP4_ENABLE approach, because it makes the diff easier to read - please update these hunks accordingly.

Reply:
While noting that ofc GitHub allows showing or hiding whitespace-only changes in diffs, the main point here is that I have copied the style of indentation used for the same change, when already made elsewhere in the commits I mentioned:

https://github.com/tianocore/edk2/commit/d933ec115bdf9be1d8dfe6a818414a14973cc0d3
https://github.com/tianocore/edk2/commit/7f17a155640a2a9e1f7b0f3522628ee2c6f62624

If I keep all the indentations left aligned (not additionally indented) in this PR, then the layout will be inconsistent with the existing changed files.

I thought (& would still try to argue...) that it is preferable to keep a consistent layout between files. Though can change if you still disagree. (Not sure to what extent this will become moot anyway, if I get the time to make the additional changes @kraxel requests.)

Reply:
For now, in the split commits, I have stuck to using the indentation introduced in https://github.com/tianocore/edk2/commit/d933ec115bdf9be1d8dfe6a818414a14973cc0d3 where I have duplicated the changes made in https://github.com/tianocore/edk2/commit/d933ec115bdf9be1d8dfe6a818414a14973cc0d3 (by @ardbiesheuvel), and using the indentation from https://github.com/tianocore/edk2/commit/7f17a155640a2a9e1f7b0f3522628ee2c6f62624 where I have duplicated the changes made in https://github.com/tianocore/edk2/commit/7f17a155640a2a9e1f7b0f3522628ee2c6f62624 (by @kraxel), even though these indentation styles are different!

Reply:
Still finishing sorting out kraxel's request. Will make the required changes here too.